src/amo/components/SearchResult/styles.scss

```diff
@@ -95,6 +95,7 @@ $icon-default-size: 32px;
   @include font-medium();
 
   color: $type-black;
+  display: flex;
```

Comment:
Since you mentioned you were having trouble with the css, you may want [`flex-shrink: 0`](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink) for the icon itself. I don't know what your exact problem was, just something worth mentioning.